### ISSUE TITLE
Bug: SQLite db - individuals should not infect themselves.

### ISSUE THREAD

**Issue by YeChen-IDM** [jonathanhhb/covid_emod_hintschoolclosure#29]
This may be related to #28, I will re-test this when #28 is fixed.

In the simulation_events.db, for NewInfection rows, I am seeing some individuals are infecting themselves. 
![image](https://user-images.githubusercontent.com/19782371/78193211-d9d2a600-742e-11ea-8ba2-11a16c4a0568.png)

**Comment by YeChen-IDM** [comment]
After reading the comments in #28, I think even if we are reporting the superinfection, we should not have somebody that infect himself. I will make sure I retest this once #28 is fixed.

**Comment by jonathanhhb** [comment]
Yes. I saw the people "infecting themselves" in the sql before I made the code change. That should not happen anymore.
